tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/embed_block.html

```diff
@@ -1 +1,33 @@
-<div class="grid__embed streamfield__embed">{{ value }}</div>
+{% load static %}
+
+<div class="grid__embed streamfield__embed youtube-video-container">
```

Comment:
Should the check for `is_youtube` come outside the div where we add the `youtube-video-container` class?

Reply:
sorting in front-end ticket.